src/animation/AnimationAction.js

```diff
@@ -21,7 +26,24 @@ THREE.AnimationAction = function ( clip, startTime, timeScale, weight, loop ) {
 	this.actionTime = - this.startTime;
 	this.clipTime = 0;
 
-	this.propertyBindings = [];
+	this.mixer = null;
+
+	var tracks = clip.tracks,
+		nTracks = tracks.length,
+		interpolants = new Array( nTracks );
```

Comment:
Interesting read!

Are you sure `new Array(n)` is actually pre-allocating `sizeof n * n`? The article I linked seems to suggest it only sets the `length` of the new array to `n`.

> var a = new Array(10);
a[0] // returns undefined
a.length // returns 10, because of reasons.